app/components/documents/batch-details/document-details-row.hbs

```diff
@@ -69,7 +70,7 @@
         >
           {{t "undo"}}
         </AuButton>
-      {{else}}
+      {{else if @row.canBeEditedOrDeleted}}
         <AuButton
           data-test-document-details-row-delete
```

Comment:
Not showing the button is fine I guess.
Would we rather show an info icon stating why? not sure how much users case about this.
Could we fall back to the book icon instead? (expand `{#if (not @isEditingEnabled)}}` with the boolean)